us.zoom.Zoom.json

```diff
@@ -40,6 +40,8 @@
                     "type": "script",
                     "dest-filename": "zoom.sh",
                     "commands": [
+                        "mkdir -p ~/config/",
+                        "test -e ~/config/zoomus.conf || ln -s /app/etc/zoomus.conf ~/config/zoomus.conf",
```

Comment:
Are you sure it reads from `~/config`? Not `~/.config`, not `~/.var/app/us.zoom.Zoom/config`?

Reply:
I guess it depends on whether it has hardcoded path instead of abiding to XDG config. Flatpak tells it that it should use ~/config. But good point, I don't know.

Reply:
`~/config` is the same directory as `~/.var/app/us.zoom.Zoom/config` only if there is `--presist=.`. Otherwise, they're not. 
`$XDG_CONFIG_HOME` is set to `~/.var/app/us.zoom.Zoom/config`.

Reply:
I'll add symlink between the two in next iteration if someone who actually has ability to test screensharing reports this doesn't work.

Reply:
What do you mean? This code is all running inside sandbox. There they are always the same, whether --persist is used or not.

Reply:
>There they are always the same, whether --persist is used or not

Nope, without any filesystem permissions, there is only `~/.var/app/$FLATPAK_ID` inside the sandbox, mounted from the host to the same path.

Reply:
You're confused. ~/.var/app/$FLATPAK_ID points to ~ *inside* the sandbox.

Reply:
No, it does so only with `--persist=.` Otherwise, `~` in sandbox is just a directly in root tmpfs.

Reply:
In either case, changing to $XDG_CONFIG_HOME should fix it, right?

Reply:
Yeah, it should be correct (if the app actually respects XDG and doesn't hardcode `~/.config`).

Reply:
About the filesystem - you can check any app without special home permissions, e.g. Telegram:
```
$ flatpak run --command=stat org.telegram.desktop --format='%d %m' /
117 /
$ flatpak run --command=stat org.telegram.desktop --format='%d %m' ~
117 /
$ flatpak run --command=stat org.telegram.desktop --format='%d %m' ~/.var/app/org.telegram.desktop
64771 /home/gasinvein/.var/app/org.telegram.desktop
$ flatpak run --command=stat org.telegram.desktop --format='%d %m' ~/Download
64771 /home/gasinvein/Download
```
Note the device id (`%d`) here - `117` is the sandboxed root tmpfs, and `64771` is my host's home partition.

Reply:
Let's have that assumption for first.